src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1260,6 +1257,16 @@ forkEnvironment.setWorkingDir("/data/folder")
 forkEnvironment.setJavaHome("/usr/java/default")
 ```
 
+By default, the forked mode is configured then all workflow tasks are executed in a forked JVM separated from the ProActive Node's JVM.
+This behavior can be controlled *globally* at the ProActive server level by modifying the global setting `pa.scheduler.task.fork` in the <<../admin/ProActiveAdminGuide.adoc#_general_configuration,scheduler configuration file>> and also *locally* at the task level via the optional checkbox property *Fork*.
+
+- In case the server is configured in non forked mode (i.e. `pa.scheduler.task.fork` = `false`), the checkbox property *Fork* is unchecked and uneditable and all the fields under *Fork Environment* are uneditable.
+
+- In case the server is configured in a forked mode (i.e. `pa.scheduler.task.fork` = `true`), the checkbox property *Fork* is checked and uneditable and all the fields under *Fork Environment* are editable.
+
+- In case the global setting is not set (i.e. `#pa.scheduler.task.fork` = `true`), each task can specify its execution mode through the checkbox property *Fork*. If the checkbox property *Fork* is unchecked, then all the other fields under *Fork Environment* are disabled.
+
+
 ==== Docker fork execution environment
 A Docker fork execution environment executes a JVM inside a Docker container. Hence the task
```

Comment:
A Docker fork execution environment is a specific usage of the fork environment mechanism. In that case, the ProActive Node's JVM starts a docker container which will run the forked JVM described above. This forked JVM can be customized by the JVM settings
and by the tool-set and isolation provided by Docker containers.